docs/HazardAnalysis/HazardAnalysis.tex

```diff
@@ -154,6 +155,44 @@ \section{Failure Mode and Effect Analysis}
       Error messages and logging & a. Allow administrator to adjust
       permissions & SR-AR1 \newline SR-AR4 & H3-2 \\
       \hline
+      Accept Contractor Document Upload & Document is valid but is uploaded in
+      the wrong location & Poor discoverability and organization &
+      a. Human-computer interface failure &User detects &
+      a. Alert administrator to adjust & FR1 & H4-1 \\
+      \cline{2-8}
+      & Document is malicious and should not be in the system & Malware
+      infection &a. Malicious user gained access to system & Error messages and
```

Comment:
error messages and logging isn't the detection. It shouldn't be allowed into the system in general and should be detected before upload

Reply:
Do you know if running a malware scan on documents prior to upload something which is possible for us to do? I see this AWS article related to it: https://aws.amazon.com/blogs/apn/integrating-amazon-s3-malware-scanning-into-your-application-workflow-with-cloud-storage-security/

Reply:
I haven't explicitly done it, but there are definitely many ways it can be done. So yeah we should do that as the detection.

Reply:
Sounds good, I'll add that in

Reply:
added "malware scan identifies malicious documents" to detection column